Notebooks/DC2Cookbook.ipynb

```diff
@@ -393,19 +393,35 @@
+    "\n",
+    "The first thing we need to make is the sky map on which the coadded images will be projected for all bands. In order to ensure that the sky map will not depend on the list of input visits (their location on the sky or on the time and place the reprocessing happens), the sky map should be absolute (using `makeSkyMap.py` and the appropriate configuration) instead of relative to an input field (as done with `makeDiscreteSkyMap.py`). Making such a sky map will create thousands of tracts and patches, that will be fixed on the sky and usable in different reprocessing sessions. \n",
+    "\n",
+    "One possibility could be to use the [Rings sky map](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42), as done for the HSC data reprocessing with the LSST stack (and currently for CFHT data reprocessing as well). In order to make this sky map, some modifications of the `lsstSimMapper.py` [script](https://github.com/lsst/obs_lsstSim/blob/master/python/lsst/obs/lsstSim/lsstSimMapper.py) must be done to match what has been done in `hasMapper.py` ([here](https://github.com/lsst/skymap/blob/master/python/lsst/skymap/ringsSkyMap.py#L42) and [here](https://github.com/lsst/obs_subaru/blob/master/python/lsst/obs/hsc/hscMapper.py#L286)). A configuration file (`makeSkyMapConfig.py` below) is then needed to run `makeSkyMap.py`, containing information on how to parametrize the rings map. An example of such a configuration can be taken from the HSC reprocessing [repository](https://github.com/LSSTDESC/ReprocessingTaskForce/blob/master/config/w_2017_49/cfht/makeSkyMapConfig.py), which contains:\n",
+    "\n",
+    "```\n",
+    "config.skyMap.name = \"rings\"\n",
+    "config.skyMap[\"rings\"].numRings = 120\n",
+    "config.skyMap[\"rings\"].projection = \"TAN\"\n",
+    "config.skyMap[\"rings\"].tractOverlap = 1.0/60 # Overlap between tracts (degrees)\n",
+    "config.skyMap[\"rings\"].pixelScale = 0.185\n",
+    "```\n",
     "\n",
-    "First we need to make the sky map.\n",
-    "Input to the sky map maker should reflect all of the data to be coadded in the future.  This is typically done by specifying the `--id` flag with no argument.  The sky map should be generated once and used by all the bands.\n",
+    "After having adapted `lsstSimMapper.py` and getting the configuration file ready, the command to run will be:\n",
+    "    \n",
     "```bash\n",
-    "$> makeDiscreteSkyMap.py $CALEXPDIR --id --output $COADDDIR\n",
-    "```"
+    "$> makeSkyMap.py INPUTDIR --output OUTPUTDIR --configfile makeSkyMapConfig.py\n",
+    "```\n",
+    "\n",
+    "The list of tracts/patches in which there is actually data (out of the 18937 tracts in that case) can be determined using a [hand-made script](https://github.com/LSSTDESC/ReprocessingTaskForce/blob/master/scripts/reportPatchesWithImages.py) developped in the context of CFHT data reprocessing (that might need some improvement)."
```

Comment:
Maybe best to bring a copy of this home-grown `reportPatchesWithImages.py` script into the ImageProcessingPipelines repo? Or is `import desc.reprocessingtaskforce` a viable command? cc @jchiang87 again.

Reply:
`import desc.reprocessingtaskforce` won't work, but we can definitely put this script in the ImageProcessingPipelines.